x-pack/test/alerting_api_integration/spaces_only/tests/alerting/builtin_alert_types/index_threshold/create_test_data.ts

```diff
@@ -8,53 +8,50 @@ import { times } from 'lodash';
 import { v4 as uuid } from 'uuid';
 import { ESTestIndexTool, ES_TEST_INDEX_NAME } from '../../../../../common/lib';
 
-// date to start writing data
-export const START_DATE = '2020-01-01T00:00:00Z';
```

Comment:
I realized when using this module again, that I had used "start date" as a parameter, but that the function writes documents from that date, and then more moving in the past.  So, it's actually an "end date".  Renamed things as appropriate, but other than that, the date logic is unchanged.

The functional change for this module is to make the number of groups written parameterized.  Previously it was always 2, now you can pass in the number of groups to create.